Sources/BraintreeCard/BTCard.m

```diff
@@ -91,14 +42,7 @@ - (NSDictionary *)parameters {
         p[@"cvv"] = self.cvv;
     }
     
-    if (self.cardholderName) {
-        p[@"cardholder_name"] = self.cardholderName;
-    }
```

Comment:
The `cardholder_name` field is already being set earlier in this method, so we removed this.